csharp/src/Ice/ObjectAdapter.cs

```diff
@@ -1193,15 +1157,24 @@ private void UpdateLocatorRegistry(LocatorInfo? locatorInfo, IObjectPrx? proxy)
             return (noProps, unknownProps);
         }
 
-        private enum State
+        private async Task PerformActivateAsync()
```

Comment:
This could be a local method of ActivateAsync

Reply:
Fixed